org.eclipse.sisu.plexus/src/main/java/org/eclipse/sisu/plexus/CompositeBeanHelper.java

```diff
@@ -264,21 +264,35 @@ private Object convertProperty( final Class<?> beanType, final Class<?> rawPrope
                                             listener );
     }
 
-    private static Method findMethod( final Class<?> beanType, final Type[] paramTypeHolder, final String methodName )
+    private Method findMethod( final Class<?> beanType, final Type[] paramTypeHolder, final String methodName,
```

Comment:
Why this was converted from class method to instance method?

Reply:
not really intentional. but could it be a problem for a private method?

Reply:
Let's undo it, created https://github.com/eclipse/sisu.plexus/pull/64